### PR TITLE
Ldanzinger/buffer fix

### PR DESCRIPTION
**Pull request by ldanzinger**
@michael-tims please review/merge

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/README.md [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
+
+Creating buffers is a core concept in GIS as it allows for proximity analysis to find geographic features contained within a polygon. For example, suppose you wanted to know how many resturants are within a short walking distance of your home. The first step in this proximity analysis would be to generate a buffer polygon of a certain distance (say 1 mile) around your house. If you are using planar buffers, make sure that the input locations and distance are suited to the spatial reference you're using. Remember that you can also create your buffers using geodesic and then project them to the spatial reference you need for display or analysis.
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/README.md]: `resturants` -> `restaurants`

File: ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/README.md [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+4. Click Clear to remove all buffers and start again.
 
 ## How it works
-The static method, `GeometryEngine.buffer`, is called by passing in a Point and a distance. The unit of measurement for the distances is automatically set to be the same as the input geometries, which in this case is meters. To provide a better user experience, the units entered in the User Interface (UI) for generating the buffer is in miles, so the provided number must be multiplied by 1609.34, which is the number of meters in a mile.
+The map point for a tap on the display is captured and the buffer distance entered (in miles) is converted to meters. The static method `GeometryEngine::buffer` is called to create a planar buffer polygon from the map location and distance. Another static method, `GeometryEngine::bufferGeodetic is called to create a geodesic buffer polygon using the same inputs. The polygon results (and tap location) are displayed in the map view with different symbols in order to highlight the difference between the buffer techniques due to the spatial reference used in the planar calculation.
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/README.md]: GeometryEngine::bufferGeodetic is missing ending `

File: ArcGISRuntimeSDKQt_QMLSamples/Geometry/Buffer/README.md [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
+
+Creating buffers is a core concept in GIS as it allows for proximity analysis to find geographic features contained within a polygon. For example, suppose you wanted to know how many resturants are within a short walking distance of your home. The first step in this proximity analysis would be to generate a buffer polygon of a certain distance (say 1 mile) around your house. If you are using planar buffers, make sure that the input locations and distance are suited to the spatial reference you're using. Remember that you can also create your buffers using geodesic and then project them to the spatial reference you need for display or analysis.
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_QMLSamples/Geometry/Buffer/README.md]: `resturants` -> `restaurants`

File: ArcGISRuntimeSDKQt_QMLSamples/Geometry/Buffer/README.md [hunk 4]
```diff
@@ -1,19 +1,27 @@
...
+4. Click Clear to remove all buffers and start again.
 
 ## How it works
-The static method, `GeometryEngine.buffer`, is called by passing in a Point and a distance. The unit of measurement for the distances is automatically set to be the same as the input geometries, which in this case is meters. To provide a better user experience, the units entered in the User Interface (UI) for generating the buffer is in miles, so the provided number must be multiplied by 1609.34, which is the number of meters in a mile.
+The map point for a tap on the display is captured and the buffer distance entered (in miles) is converted to meters. The static method `GeometryEngine.buffer` is called to create a planar buffer polygon from the map location and distance. Another static method, `GeometryEngine.bufferGeodetic is called to create a geodesic buffer polygon using the same inputs. The polygon results (and tap location) are displayed in the map view with different symbols in order to highlight the difference between the buffer techniques due to the spatial reference used in the planar calculation.
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_QMLSamples/Geometry/Buffer/README.md]: GeometryEngine.bufferGeodetic is missing the ending `

**Comment by ldanzinger** [comment]
@michael-tims updated

**Pull request merged**